frontend/src/index.tsx

```diff
@@ -71,6 +72,10 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
                                             path: AppRoute.FORGOT_PASSWORD,
                                             element: <Auth />,
                                         },
+                                        {
+                                            path: AppRoute.QA,
```

Comment:
QA route works only for not logged in users. Shouldn't it be for logged in users? Because the only link we have to QA page is in the menu for logged in user and it will not work with the current implementation.

Reply:
Done